Formula/urh.rb

```diff
@@ -31,8 +31,13 @@ class Urh < Formula
   end
 
   def install
-    site_packages = Language::Python.site_packages("python3")
-    ENV.prepend_create_path "PYTHONPATH", Formula["libcython"].opt_libexec/site_packages
+    # Enable finding cython, which is keg-only
+    site_packages = Language::Python.site_packages("python3.10")
```

Comment:
Suggest including a `def python3 ... end` per the recommendation of https://github.com/Homebrew/homebrew-core/issues/108008 and updating the `test do` block as well

Reply:
Went the first form `python3 = "python3.10"` since it's only referenced in the install block. And I tried to add a test, but `urh` just launches a GUI and I couldn't find any flags to change that (beyond `--version`, which doesn't catch this bug)